### PR TITLE
Pulled prismic data for author info

### LINKED ISSUE #9: Pull Data: Author Info

### PR DESCRIPTION
**Pull request by jzhzhang66**
- Pulled initial prismic data for AuthorInfo component 
- Added a function call to retrieve contact page data in api.js (getAuthorInfo)
- Renders author info on story page 
Screenshot 2021-03-14 204048

### CODE REVIEW

File: components/AuthorInfo.js [hunk 1]
```diff
@@ -1,5 +1,23 @@
...
+
+    return (
+        <>
+        {renderRichText(authorName)}
+        {renderRichText(cameraInfo)}
+        <img src={signature.url} alt="" />
+        <a>Signature dimensions: {signature.dimensions.width} x {signature.dimensions.height} </a>
+        <br /> <br />
+        <img src={authorImage.url} alt="" />
+        <a>Author image dimensions: {authorImage.dimensions.width} x {authorImage.dimensions.height} </a>
```
Comment (isabellberzsenyi) [inline components/AuthorInfo.js]: Styling comment: let's indent these lines in one level from the <> </>

**Review by isabellberzsenyi — COMMENTED** [review]
Looks good other than a little styling comment. Also, for the future if you find that a commit is undoing something from a previous commit you can undo commits and git force push. I can show you if needed

File: components/AuthorInfo.js (hunk 1 above)
Comment (josephmannis) [reply components/AuthorInfo.js]: Follow up here, creating a .prettier config may help!

**Review by josephmannis — CHANGES_REQUESTED** [review]
Looks good, but can we pull in alt text for these images?

File: components/AuthorInfo.js (hunk 1 above)
Comment (jzhzhang66) [reply components/AuthorInfo.js]: Thanks, just fixed it! And will look into a .prettier config this week!

**Review by isabellberzsenyi — APPROVED** [review]
✨

**Review by jean-zhang — APPROVED** [review]
LGTM!

**Pull request merged**